.gitignore

```diff
@@ -34,3 +34,5 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+./src/config.ts
```

Comment:
I don't wanna add the config file to `.gitignore` just yet, as I need to move some variables like `revalidateTime` to actual internal files.